back-end/src/main/java/admin/worker/application/WorkerService.java

```diff
@@ -69,11 +77,19 @@ private JobResponse findFirstWaitingJob(Long gpuBoardId) {
 
     private GpuBoard findGpuBoardByGpuServerId(Long serverId) {
         return gpuBoardRepository.findByGpuServerId(serverId)
-                .orElseThrow(GpuBoardException.GPU_BOARD_NOT_FOUND::getException);
+            .orElseThrow(GpuBoardException.GPU_BOARD_NOT_FOUND::getException);
     }
 
     private GpuServer findGpuServerById(Long serverId) {
         return serverRepository.findById(serverId)
-                .orElseThrow(GpuServerException.GPU_SERVER_NOT_FOUND::getException);
+            .orElseThrow(GpuServerException.GPU_SERVER_NOT_FOUND::getException);
+    }
+
+    public Long saveLog(Long jobId, WorkerJobLogRequest workerJobLogRequest) {
```

Comment:
```suggestion
    @Transactional
    public Long saveLog(Long jobId, WorkerJobLogRequest workerJobLogRequest) {
```
트랜잭셔널을 붙여아 JPA가 동작할 것 같다는 의견인데.. 혹시 잘못알고 있는 거라면 알려주세용!ㅎㅎㅎ

Reply:
그렇네요 😱
아래 코드에서 사용하도록 빠르게 수정하겠습니다. 👍

Reply:
`@Transactional`을 붙이지 않아도 동작 합니다. 제가 유추하기로는 JPA 내부 로직에서 `@Tranactional`을 붙여주는 것 같아요.

Reply:
오 이거 에드랑 같이 까봤던 코드네요!!
JpaRepository에서 기본 CRUD에 @Tranactional을 먹였더라고요.

다만 이렇게하면 각 db접근에, 
즉 jobRepository.find와 logRepository.find 각각에서 다른 entityManger가 열리고 닫히게 되는게 아닌가,

그냥 @Transactional으로 하면 WorkerService의 save 전체가 트랜젝션 안에서 관리됨이 확실해지는데, 혹시 안붙이는 이유가 따로 있는지 궁금하네요.

제가 JPA를 제대로 몰라서, 이렇게 따로 트랜젝션을 열었을 때 어떤 장점을 갖는지 모르겠습니다!!

Reply:
여기서 save 연산만 하는 줄 알고 배럴이 동작하는지 물어봐서
JPA를 사용할 떄는 하나의 연산이더라도 `@Transactional` 붙여야하지 않냐고 물어본 줄 알았네요. 😅

두 가지 연산(findById와 save)를 묶어서 하니 `@Transactional` 붙이는게 맞네요.
고치겠습니다. 👍